website/layout/AutodocsLayout.js

```diff
@@ -504,47 +543,4 @@ var Autodocs = React.createClass({
   }
 });
 
-var EmbeddedSimulator = React.createClass({
```

Comment:
Sorry for the trash in the diff, I moved those two classes above. It's annoying to have the main component not be the last.